Sources/Host/Linux/ARM/PTraceARM.cpp

```diff
@@ -28,50 +28,9 @@ namespace ds2 {
 namespace Host {
 namespace Linux {
 
-struct PTracePrivateData {
-  uint8_t breakpointCount;
-  uint8_t watchpointCount;
-  uint8_t maxWatchpointSize;
+void PTrace::initCPUState(ProcessId pid) {}
```

Comment:
Ah, you removed these only from the ARM code, that's why. Any reason why we're keeping around this PrivateData stuff for other architectures?